src/gui/application.cpp

```diff
@@ -643,7 +647,19 @@ void Application::parseOptions(const QStringList &options)
         } else if (option.endsWith(QStringLiteral(APPLICATION_DOTVIRTUALFILE_SUFFIX))) {
             // virtual file, open it after the Folder were created (if the app is not terminated)
             QTimer::singleShot(0, this, [this, option] { openVirtualFile(option); });
-        } else {
+        } else if (option.startsWith(QStringLiteral(APPLICATION_URI_HANDLER_SCHEME "://open"))) {
+            // the URL must be 'APPLICATION_URI_HANDLER_SCHEME://open/username@example.com<optional port number :8080>/relative-path-to-file'
+            // e.g. nc://open/admin@nextcloud.lan:8080/Photos/lovely.jpg
+            // e.g. nc://open/user@example.de/Photos/lovely.jpg
```

Comment:
maybe it would be a good idea to document that feature into our user documentation
or at least include a link to such a doc if it already exists in the server documentation
code comments are not an easy way to discover new features

Reply:
@mgallien But these are meant for developers, not for users. User must not care about the URL format as it is the server that generates the URL.

Reply:
@mgallien added it to a user documentation, see last commit